### PR TITLE
Enrich additional data in ProjectCreated.

### PR DESCRIPTION
**Pull request by jnaviask**
Removes requirement from Commonwealth to fetch project contract information.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
   },
   "homepage": "https://github.com/hicommonwealth/chain-events#readme",
   "dependencies": {
+    "@commonwealth/chain-events": "file:.yalc/@commonwealth/chain-events",
```
Comment (jnaviask) [inline package.json:37]: We should remove this / figure out why this exists.

File: src/chains/commonwealth/subscribeFunc.ts [hunk 2]
```diff
@@ -50,7 +50,7 @@ export async function createApi(
 
       // create subscriptions for all existing projects
       const nProjects = await api.factory.numProjects();
-      for (let projectN = 1; projectN <= nProjects.toNumber(); projectN++) {
+      for (let projectN = 1; projectN <= nProjects; projectN++) {
```
Comment (jnaviask) [inline src/chains/commonwealth/subscribeFunc.ts:53]: Why remove the toNumber?

**Review by jnaviask — COMMENTED** [review]
Few minor comments. Did not QA. Looks good.

File: src/chains/commonwealth/subscribeFunc.ts (hunk 2 above)
Comment (dantrinh) [reply src/chains/commonwealth/subscribeFunc.ts:53]: removed because numProjects is not a string anymore but a number in the Contract

Comment (jnaviask) [reply src/chains/commonwealth/subscribeFunc.ts:53]: Numbers in contracts are typically returned as BigNumber so we store them as strings, unless we can verify that a JS number has sufficient precision to represent it.

File: package.json (hunk 1 above)
Comment (dantrinh) [reply package.json]: Removed!

**Review by dantrinh — APPROVED** [review]

**Pull request merged**